uber/templates/preregistration/preregbase.html

```diff
@@ -64,12 +66,12 @@
             row: '#badge-types',
             selector: '.badge-type-selector',
             options: [{
```

Comment:
I'd greatly prefer that we use the `ribbon_and_or_badge` property for this, as it's exactly what I made it for. That's an easy fix to make later, though.